pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
I think we need to do this because of `/share/applications` and other paths that we would want to see in the user environment.